### PR TITLE
Change my name, change my name

### PR DESCRIPTION
**Pull request by aaronschachter**
#### What's this PR do?

Finds and replaces all instances of `Freddie` with `Tej`. I'll be adding a few comments per specific changes upon opening this PR.

#### How should this be reviewed?
* Text JOIN or LESS and confirm `Tej` is returned
* Text "Hi there" and confirm`Tej` is returned

Example:
```
You> hi there

Bot> Hi! This is Tej from DoSomething.org! I text about cool ways to take action in your community, AND scholarship opportunities. Go to DoSomething.org to see!
```

#### Any background context you want to provide?

Although we can override the welcome compliance text via config variable #487, it makes sense to change our hardcoded default, and merge/deploy when we're ready to make the switch.

#### Relevant tickets

https://dosomething.slack.com/archives/C2C8NLNAY/p1553707505112700?thread_ts=1553697026.111500&cid=C2C8NLNAY

#### Checklist
- [ ] Documentation added for new features/changed endpoints.
- [ ] Tested on staging.

### CODE REVIEW

File: brain/index.rive [hunk 1]
```diff
@@ -3,7 +3,7 @@
 // Bot variables
 ! var cmd_support = Q
 ! var member_count = 6 MILLION+
-! var name = Freddie
+! var name = Tej
```
Comment (aaronschachter) [inline brain/index.rive]: This provides a `<bot name>` [Rivescript variable](https://www.rivescript.com/docs/tutorial#definitions), which gets used in an [`faqAnswer`](https://app.contentful.com/spaces/owik07lyerdj/entries/30ZJUQPlIu8BR5iXTfvzEA) that a few triggers link to:
- `who is @messaging [me]`
- `who [*] is @botname`

File: brain/index.rive [hunk 2]
```diff
@@ -63,7 +63,7 @@
 ! array alone     = alone|me|myself|self|solo
 ! array bike      = bicycle|bike|bikin|biking|cycling
 ! array bot       = bot robot machine computer
-! array botname   = freddie freddi alicia alysa alysha
+! array botname   = freddie freddi alicia alysa alysha tej
```
Comment (aaronschachter) [inline brain/index.rive]: Per https://github.com/DoSomething/gambit/pull/491/files#r269783896, this defines a `@botname` substitution. This particular trigger `who is Freddie/Freddi/etc` was ported over from our Mobile Commons keywords, so we might not even need it and this substitution at all anymore... but keeping in for sake of less changes (and also documentation).

**Comment by aaronschachter** [comment]
Probably makes sense to wait on merging this until we're ready to deploy. @ngjo Do we need to wait until Monday morning, or could this happen EOD Friday?

**Comment by ngjo** [comment]
> Do we need to wait until Monday morning, or could this happen EOD Friday?
I think it'll be best to wait until Tuesday morning actually since Freddie's still here today and I think promotions of their goodbye content/text is still being wrapped up. They're intro'ing Tej on Tuesday too!

**Pull request merged**